### PR TITLE
fix torch.nn.BatchNorm

### PR DESCRIPTION
**Pull request by ZS-YANG**
…torch.nn.BatchNorm

### CODE REVIEW

**Comment by ZS-YANG** [comment]
```
torch描述
track_running_stats: a boolean value that 
when set to ``True``, this  module tracks the running mean and variance, and 
when set to ``False``,   this module does not track such statistics, and initializes statistics buffers :attr:`running_mean` and :attr:`running_var` as ``None``.            
When these buffers are ``None``, this module always uses batch statistics.  in both training and eval modes. Default: ``True``
```
```
paddle描述
若设置为 False，则使用一个 mini-batch 的统计数据。
若设置为 True 时，将使用全局统计数据。
若设置为 None，则会在测试阶段使用全局统计数据，在训练阶段使用一个 mini-batch 的统计数据。默认值为 None。
```

File: docs/guides/model_convert/convert_from_pytorch/api_difference/nn/torch.nn.BatchNorm1d.md [hunk 1]
```diff
@@ -29,11 +29,11 @@ paddle.nn.BatchNorm1D(num_features,
 | num_features           | num_features      | 表示输入 Tensor 通道数。                                     |
 | eps           | epsilon      | 为了数值稳定加在分母上的值，仅参数名不一致。                                     |
 | momentum           | momentum      | 表示归一化函数中的超参数, PyTorch 和 Paddle 公式实现细节不一致，两者正好是相反的，需要转写。                                     |
+| affine              | -                | 是否进行反射变换， Paddle 无此参数，需要转写。                                                                                         |
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/nn/torch.nn.BatchNorm1d.md]: 顺序的话原来那样也可以，不用调整

File: docs/guides/model_convert/convert_from_pytorch/api_difference/nn/torch.nn.BatchNorm1d.md [hunk 2]
```diff
@@ -29,11 +29,11 @@ paddle.nn.BatchNorm1D(num_features,
...
+| track_running_stats | use_global_stats | 指示是否使用全局均值和方差，Pytorch 设置为 True，Paddle 需设置为 True；Pytorch 设置为 None，Paddle 需设置为 False；Pytorch 设置为 False，Paddle 需设置为 False，需要转写。 |
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/nn/torch.nn.BatchNorm1d.md]: 看下这个转换规则的代码逻辑，这个描述可能不对

https://github.com/PaddlePaddle/PaConvert/blob/master/paconvert/api_matcher.py#L1009-L1022

**Review by zhwesky2010 — COMMENTED** [review]
基于代码再分析一下，必要时需要对比计算结果才可确认

File: docs/guides/model_convert/convert_from_pytorch/api_difference/nn/torch.nn.BatchNorm1d.md (hunk 1 above)
Comment (ZS-YANG) [reply docs/guides/model_convert/convert_from_pytorch/api_difference/nn/torch.nn.BatchNorm1d.md]: done

File: docs/guides/model_convert/convert_from_pytorch/api_difference/nn/torch.nn.BatchNorm1d.md (hunk 2 above)
Comment (ZS-YANG) [reply docs/guides/model_convert/convert_from_pytorch/api_difference/nn/torch.nn.BatchNorm1d.md]: done

**Comment by ZS-YANG** [comment]
pytorch https://github.com/pytorch/pytorch/blob/162109f6c2e852827b36eeb95a72f7d396682ee4/torch/nn/modules/batchnorm.py#L344

paddle
https://github.com/PaddlePaddle/Paddle/blob/develop/python/paddle/nn/layer/norm.py#L1285

**Review by zhwesky2010 — APPROVED** [review]
LGTM
